tests/integration/standalone/override-install.nix

```diff
@@ -0,0 +1,6 @@
+{ hmInstall ? (import <home-manager> { }).install }:
+hmInstall.override {
+  shellHook = ''
+    exec home-manager init --switch --no-flake -b backup
```

Comment:
I'm surprised this works, looking at the code I don't see any handling of `-b` in the `doInit` function. It may work (mostly accidentally) if the `-b backup` is placed before the `init` sub-command.